interface/themes/core.css

```diff
@@ -89,14 +89,22 @@ button[href*="deleter"]:before {
 .btn-undo::before {
     content: '\f0e2\00a0\00a0';
 }
+.btn-update::before {
```

Comment:
TODO:
There is already an update button in this script. Which one to use? @zbig01

Reply:
FIXED